src/style.css

```diff
@@ -1,3 +1,79 @@
+:root {
+  --navhover-color: #00acdc;
+  --header-height: 60px;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav-list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav-item {
+  display: flex;
+}
+
+.nav-item + .nav-item {
+  margin-left: 20px;
+}
+
+.nav-link {
+  position: relative;
+  text-decoration: none;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-align: center;
+  height: var(--header-height);
+  line-height: var(--header-height);
+}
+
+.nav-link:link {
+  color: #000;
+}
+
+.nav-link:hover {
+  color: var(--navhover-color);
+}
+
+.nav-link.is-active {
+  color: var(--navhover-color);
+  display: block;
+}
+
+.nav-link.is-active::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--navhover-color);
+  border-radius: 8px;
```

Comment:
The `border-radius` property on the `::before` pseudo-element might not be necessary if the intention is to create a straight line indicator. Consider removing it unless a rounded line is desired.